mobilelib.php

```diff
@@ -784,10 +787,14 @@ function hsuforum_mobile_add_filteredpost_to_posts_array(&$rootposts, $rootposti
         $rootposts[$rootpostid]->filteredposts = [];
         array_push($rootposts[$rootpostid]->filteredposts, $postid);
     }
+
     // Add likes for sorting options
-    $postlikes = sizeof($DB->get_records('hsuforum_actions', array('postid' => $postid, 'action' => 'like')));
-    if ($postlikes) {
-        $rootposts[$rootpostid]->rootpostlikes += $postlikes;
+    if (count($postlikes) > 0) {
```

Comment:
This can be turned into a guard clause if you turn around the comparison to reduce nesting ifs.